UsefulProteomicsDatabases/PtmListLoader.cs

```
@@ -127,43 +128,56 @@ private static IEnumerable<Modification> ReadMod(List<string> specification, Dic
             {
                 if (line.Length >= 2)
                 {
-                    switch (line.Substring(0, 2))
+                    string modKey = line.Substring(0, 2);
+                    string modValue = null;
+                    if (line.Length > 5)
+                        try
+                        {
+                            modValue = line.Split('#')[0].Trim().Substring(5);
+                        }
+                        catch
```

Why is there a try catch here? Try to use a boolean...

---

Nevermind, just put a comment noting the intent of the empty catch block.